docs/UserGuide.md

```diff
@@ -2,133 +2,411 @@
-   
-1. Download the latest `addressbook.jar` from the [releases](../../../releases) tab.
-2. Copy the file to the folder you want to use as the home folder for your Address Book.
-3. Double-click the file to start the app. The GUI should appear in a few seconds. 
-   > <img src="images/Ui.png" width="600">
-
-4. Type the command in the command box and press <kbd>Enter</kbd> to execute it. <br>
-   e.g. typing **`help`** and pressing <kbd>Enter</kbd> will open the help window. 
-5. Some example commands you can try:
-   * **`list`** : lists all contacts
-   * **`add`**` John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01` : 
-     adds a contact named `John Doe` to the Address Book.
-   * **`delete`**` 3` : deletes the 3rd contact shown in the current list
-   * **`exit`** : exits the app
-6. Refer to the [Features](#features) section below for details of each command.<br>
+1. Ensure you have Java version `1.8.0_60` or later installed in your computer.
 
+  > Having any Java 8 version is not enough.
+  > This app will not work with earlier versions of Java 8.
+
+2. Download the latest `TaskTracker.jar` from the 'releases' tab.
+
+3. Copy the file to the folder you want to use as the home folder for your task
+   database.
+
+4. Double-click the file to start the app. The GUI should appear in a few
+   seconds.
+
+5. Set up: enter your name and verify the curent date and time.
+
+6. You're good to go! Try adding your first Task. press ';' to launch/hide the
+   command bar. Press Enter to enter a command. For floating task, `add
+   [taskname]`!
+
+   ![alt text](./images/userguide/command_bar.png "command bar")
+
+7. Add task with due date/time, example commands are:
+
+	* `add "bake cookies" 31/12 1500`<br>
+      bake cookies by 31 December of this year, 3pm.
+
+	* `add "bake cookies" 1500`<br>
+      bake cookies by 3pm today.
+
+   ![alt text](./images/userguide/task.png "")
+
+8. Add event. An event is a task with a start time and end time, example
+   commands are:
+
+	* `add "CS2103t summer camp" 30/8 1300 3/1/2017 1800`
+
+      Event CS2103t summer camp starts on 30 Aug 2016, 1pm, and ends on 3
+      January 2017,6pm.
+
+	* `add "potato peeling" 1500 1900`
+
+      Event potato peeling starts today 3pm and ends today 7pm.
+
+	* `add "cupcake festival" 30/08 800 1200`
+
+      Event cupcake festival starts on 8am and ends on 12pm of 24 Oct of this
+      year.
+
+	* `add "Trick or treat" 800 31/10 900`
+
+      Event Trick or treat starts today 8pm and ends on 31 Oct 9pm.
+
+   The events for current selected day will show up under schedule.
+
+   ![alt text](./images/userguide/event.png "")
+
+9. To see the schedule of a different day, press 'C' to toggle calender mode on
+   and off. Use arrow keys and enter to select the day to view. The schedule
+   for the day will show up in the schedule column.
+
+   ![alt text](./images/userguide/calender.png "")
+
+   ![alt text](./images/userguide/toggle_calender.png "")
```

Comment:
make sure you've thought through the UI:
* How are the deadline tasks ordered? since it looks like earlier tasks are displayed later here
* Can make better use of colours to distinguish between components 
* Can I switch panes/scroll using the keyboard? 
* The events look hard to read in this pic
* Is it necessary to hide the command bar? 
* Is there feedback for users when they enter commands? If so, where is it displayed?